src/net/java/sip/communicator/plugin/loggingutils/LoggingConfigForm.java

```diff
@@ -246,7 +247,7 @@ private void init()
         c.gridy = 4;
         mainPanel.add(archiveButton, c);
 
-        if(!StringUtils.isNullOrEmpty(getUploadLocation()))
+        if(StringUtils.isEmpty(getUploadLocation()))
```

Comment:
isNotEmpty

Reply:
Thanks! I tried to review it by my self before commiting for catching the similar issues. But seems I miss one :-)